src/app/admin/components/DataTableSectionUser.tsx

```diff
@@ -0,0 +1,381 @@
+import Image from "next/image";
+import { useEffect, useMemo, useState } from "react";
+import { Controller, type SubmitHandler, useForm } from "react-hook-form";
+import { Bounce, toast } from "react-toastify";
+
+import { type Schema } from "@/amplify/data/resource";
+import PopupUser from "@/app/admin/components/PopupTileUser";
```

Comment:
Just a small note: When importing a React Component, make sure to use the name of the component. In this case it should be `PopupTileUser`

CC: @nataliey16 @burtonjong